src/webdriverio-tester.js

```diff
@@ -0,0 +1,192 @@
+/**
+ * @author Adam Meadows [@job13er](https://github.com/job13er)
+ * @copyright 2015 Cyan, Inc. All rights reserved
+ */
+
+'use strict';
+
+/**
+ * @typedef Result
+ * @property {Number} code - the exit code of the command
+ * @property {String} stdout - the standard output from command
+ */
+
+var _ = require('lodash');
+var path = require('path');
+var sh = require('execSync');
```

Comment:
I would prefer to use builtin async methods as well. Actually, I'd rather use the `http` module in `node` vs. using `exec` at all. Then we don't need `curl` to be installed. However, with any async, I don't know how to tell the command line utility to not exit until receiving a response (or timing out). 

Do you have any ideas for how to handle exit codes when using async?